升级至 2.10.0 之后，插件会导致报错

升级 2.10.0 之后，使用此插件会导致 Halo 报错，报错信息如下所示：

```
Caused by: java.lang.ClassCastException: class com.fasterxml.jackson.databind.node.ObjectNode cannot be cast to class com.fasterxml.jackson.databind.JsonNode (com.fasterxml.jackson.databind.node.ObjectNode is in unnamed module of loader org.springframework.boot.loader.LaunchedURLClassLoader @87aac27; com.fasterxml.jackson.databind.JsonNode is in unnamed module of loader org.pf4j.PluginClassLoader @742ba594)
	at java.base/java.lang.Iterable.forEach(Unknown Source) ~[na:na]
```

--- a/.github/workflows/workflow.yaml
+++ b/.github/workflows/workflow.yaml
@@ -17,8 +17,8 @@ jobs:
       - name: Set up JDK 17
         uses: actions/setup-java@v2
         with:
-          distribution: 'temurin'
-          cache: 'gradle'
+          distribution: "temurin"
+          cache: "gradle"
           java-version: 17
       - name: Build with Gradle
         run: |
@@ -27,12 +27,12 @@ jobs:
           version=${version#v}
           sed -i "s/version=.*-SNAPSHOT$/version=$version/1" gradle.properties
           ./gradlew clean build -x test
-      - name: Archive plugin-starter jar
+      - name: Archive plugin-live2d jar
         uses: actions/upload-artifact@v2
         with:
-          name: plugin-starter
+          name: plugin-live2d
           path: |
-            build/libs/*-plain.jar
+            build/libs/*.jar
           retention-days: 1
 
   github-release:
@@ -40,10 +40,10 @@ jobs:
     needs: build
     if: github.event_name == 'release'
     steps:
-      - name: Download plugin-starter jar
+      - name: Download plugin-live2d jar
         uses: actions/download-artifact@v2
         with:
-          name: plugin-starter
+          name: plugin-live2d
           path: build/libs
       - name: Get Name of Artifact
         id: get_artifact
@@ -79,3 +79,26 @@ jobs:
               name: artifactName,
               data: await fs.readFile(artifactPathName)
             });
+
+  app-store-release:
+    runs-on: ubuntu-latest
+    needs: build
+    if: github.event_name == 'release'
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: true
+      - name: Download plugin-live2d jar
+        uses: actions/download-artifact@v2
+        with:
+          name: plugin-live2d
+          path: build/libs
+      - name: Sync to Halo App Store
+        uses: halo-sigs/app-store-release-action@main
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          app-id: ${{secrets.APP_ID}}
+          release-id: ${{ github.event.release.id }}
+          assets-dir: "build/libs"
+          halo-username: ${{ secrets.HALO_USERNAME }}
+          halo-password: ${{ secrets.HALO_PASSWORD }}

--- a/gradle.properties
+++ b/gradle.properties
@@ -1,1 +1,1 @@
-version=1.1.2
+version=1.1.3-SNAPSHOT

--- a/src/main/resources/plugin.yaml
+++ b/src/main/resources/plugin.yaml
@@ -4,7 +4,6 @@ metadata:
   name: PluginLive2d
 spec:
   enabled: true
-  version: 1.1.2
   requires: ">=2.4.0"
   author:
     name: LIlGG

